core/quivr_core/processor/registry.py

```diff
@@ -119,32 +120,32 @@ def defaults_to_proc_entries(
                 priority=None,
             )
 
-    # TODO(@aminediro): Megaparse should register itself
-    # Append Megaparse
-    _append_proc_mapping(
-        mapping=base_processors,
-        file_exts=[
-            FileExtension.pdf,
-            FileExtension.docx,
-            FileExtension.doc,
-            FileExtension.pptx,
-            FileExtension.xls,
-            FileExtension.xlsx,
-            FileExtension.csv,
-            FileExtension.epub,
-            FileExtension.bib,
-            FileExtension.odt,
-            FileExtension.html,
-            FileExtension.py,
-            FileExtension.markdown,
-            FileExtension.md,
-            FileExtension.mdx,
-            FileExtension.ipynb,
-        ],
-        cls_mod="quivr_core.processor.implementations.megaparse_processor.MegaparseProcessor",
-        errtxt=f"can't import MegaparseProcessor. Please install quivr-core[{ext_str}] to access MegaparseProcessor",
-        priority=None,
-    )
+    # TODO(@aminediro @chloedia): Megaparse should register itself
+    if getenv("MEGAPARSE_URL") and getenv("MEGAPARSE_API_KEY"):
```

Comment:
Tha aws url for megaparse is defaulted in MegaParseSDK, so you just need to check API and even then, what if we want to run megaparse API locally -> we force api key?